packages/commons/src/auth/authData.ts

```diff
@@ -37,7 +37,7 @@ const CommaSeparatedStringToArray = <T extends z.ZodType>(t: T) =>
 const SharedStandardJWTClaims = z.object({
   // All standard claims except "sub", which is not present in UI tokens
   iss: z.string(),
-  aud: CommaSeparatedStringToArray(z.string()),
+  aud: z.union([z.array(z.string()), CommaSeparatedStringToArray(z.string())]),
```

Comment:
This is because the token generator copied from https://github.com/pagopa/interop-be-reports/tree/1.0.x/jobs/commons/src/services generates tokens with `aud` as a JSON array in the JSON token payload, while so far we supported only a comma-separated string of values

Reply:
let's keep it like this for now, but in the RFC the `aud` field i defined as an array, with the possible only exception of being a single value when there is only one audience. 
No references to comma separated strings, so I wonder where we used this approach

Reply:
The example tokens provided in https://pagopa.atlassian.net/browse/IMN-363 had comma-separated strings as values for `"aud"` so I based the implementation of `authData` on that. However, I don't know if these tokens were mocks or were generated from a part of the existing codebase...

Currently, the services already in production don't accept arrays passed as values for `"aud"` - since everything if working, either they are being called only with single string values or some part of the existing codebase generates comma-separated audiences

Reply:
yup I agree, we will investigate this behaviour but being outside the scope of this PR IMO right now your change is fine 👌